### PR TITLE
[zh] Resync configure service account page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by tengqm** [comment]
@sftim This resync is really not fun. This heavy work was largely introduced by 0fda59f8df0017f4b8f67047423ebb74dcf9b5d9.

File: content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -277,42 +405,60 @@ token:          ...
...
-### Create an imagePullSecret
+<!--
+## Add ImagePullSecrets to a service account
 
-- Create an imagePullSecret, as described in [Specifying ImagePullSecrets on a Pod](/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod).
+First, [create an imagePullSecret](/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod).
+Next, verify it has been created. For example:
 -->
-## 为服务账号添加 ImagePullSecrets  {#add-imagepullsecrets-to-a-service-account}
+## 为服务账号添加 ImagePullSecret    {#add-imagepullsecrets-to-a-service-account}
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md:431]:
```suggestion
## 为服务账号添加 imagePullSecrets    {#add-imagepullsecrets-to-a-service-account}
```
Two code snippets below have this field:
```sh
kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "myregistrykey"}]}'
```
```yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  creationTimestamp: 2015-08-07T22:02:39Z
  creationTimestamp: 2021-07-07T22:02:39Z
  name: default
  namespace: default
  uid: 052fb0f4-3d50-11e5-b066-42010af0d7b6
imagePullSecrets:
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md [hunk 2]
```diff
@@ -322,82 +468,74 @@ The content of `token` is elided here.
...
 <!--
-### Verify imagePullSecrets was added to pod spec
+### Verify that imagePullSecrets are set for new Pods
 
-Now, when a new Pod is created in the current namespace and using the default ServiceAccount, the new Pod has its  `spec.imagePullSecrets` field set automatically:
+Now, when a new Pod is created in the current namespace and using the default
+ServiceAccount, the new Pod has its `spec.imagePullSecrets` field set automatically:
 -->
-### 验证镜像拉取 Secret 已经被添加到 Pod 规约
+### 检查 imagePullSecret 已经被设置到新 Pod 上  {#verify-that-imagepullsecrets-are-set-for-new-pods}
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md:535]:
```suggestion
### 检查 imagePullSecrets 已经被设置到新 Pod 上  {#verify-that-imagepullsecrets-are-set-for-new-pods}
```

**Review by windsonsea — APPROVED** [review]
/lgtm

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**